.travis.yml

```diff
@@ -1,19 +1,19 @@
 os: linux
-dist: trusty
-
-before_install:
-    - sudo apt-get -qq update
-    - sudo apt-get install -y libdbus-glib-1-dev
-
 language: python
-python: "3.4"
+
+addons:
+  apt:
+    packages:
+      libdbus-glib-1-dev
 
 branches:
     only: master
 
 matrix:
     include:
         - env: TASK=lint
+          python: "3.4" # dbus-python does not compile on higher versions
```

Comment:
Please elaborate on this.  AFAIK `dbus-python` appears to work fine on my python 3.7.2 system?

Reply:
It fails on Travis, which is running Ubuntu. You can look at examples of how it fails here:
https://travis-ci.org/stratis-storage/into-dbus-python/builds/494037710.

Reply:
Note that this is the same decision, here: https://github.com/stratis-storage/stratisd/pull/1436.

Reply:
Is there some reason we can't use a package install instead of building from source?  All we are trying to do is lint the code.

Reply:
What do you mean by a package install?

Reply:
> What do you mean by a package install?

I mean satisfy a python library dependency by using a package install ref. 
https://packages.ubuntu.com/trusty/python3-dbus
instead of building dbus-python from source.

You are using a package install to satisfy the dependency of `libdbus-glib-1-dev`

Reply:
Yes. But from that link you gave me:

```
python3 (<< 3.5)
    interactive high-level object-oriented language (default python3 version)
```

So, I think it's the same either way.